.github/workflows/tests-ces-migrate.yaml

```
@@ -0,0 +1,240 @@
+name: CiliumEndpointSlice migration (ci-ces-migrate)
+
+# Any change in triggers needs to be reflected in the concurrency group.
+on:
+  pull_request: {}
+  push:
+    branches:
+      - main
+      - ft/main/**
```

```suggestion
  push:
    branches:
      - main
      - ft/main/**
    paths-ignore:
      - 'Documentation/**'
      - 'test/**'
```
If you add the paths to ignore here and use ariane for PR trigger, you can drop the `check_changes` step below